src/components/rules-creation-page/steps/IncludeExcludeSelectionStep.tsx

```diff
@@ -0,0 +1,236 @@
+    dataElementGroupSets: "Data element group sets",
+    "dataElementGroupSets.attributes": "Attributes of data element group sets",
+    indicators: "Indicators",
+    "indicators.attributes": "Attributes of indicators",
+    "indicators.indicatorGroups": "Indicator groups of indicators",
+    indicatorGroups: "Indicator groups",
+    "indicatorGroups.attributes": "Attributes of indicator groups",
+    "indicatorGroups.indicators": "Indicators of indicator groups",
+    "indicatorGroups.indicators.attributes": "Attributes of indicators in indicator groups",
+    "indicatorGroups.indicators.indicatorGroups":
+        "Indicator groups of indicators in indicator groups",
+    "indicatorGroups.indicatorGroupSet": "Indicator group sets of indicator groups",
+    "indicatorGroups.indicatorGroupSets": "Indicator group sets of indicator groups",
+    indicatorGroupSets: "Indicator group sets",
+    "indicatorGroupSets.attributes": "Attributes of indicator group sets",
+    indicatorType: "Indicator type",
+    legendSets: "Legend sets",
+    programs: "Programs",
+    optionSets: "Option sets",
+    "optionSets.options": "Options in option sets",
+    organisationUnits: "Organisation units",
+    "organisationUnits.attributes": "Attributes of organisation units",
+    "organisationUnits.organisationUnitGroups": "Organisation unit groups of organisation units",
+    organisationUnitGroups: "Organisation unit groups",
+    "organisationUnitGroups.attributes": "Attributes of organisation unit groups",
+    "organisationUnitGroups.organisationUnits": "Organisation units in organisation unit groups",
+    "organisationUnitGroups.organisationUnits.attributes":
+        "Attributes of organisation units in organisation unit groups",
+    "organisationUnitGroups.organisationUnits.organisationUnitGroups":
+        "Organisation unit groups of organisation units in organisation unit groups",
+    "organisationUnitGroups.organisationUnitGroupSets":
+        "Organisation unit group sets of organisation unit groups",
+    "organisationUnitGroups.organisationUnitGroupSets.attributes":
+        "Attributes of organisation unit group sets in organisation unit groups",
+    organisationUnitGroupSets: "Organisation unit group sets",
+    "organisationUnitGroupSets.attributes": "Attributes of organisation unit group sets",
+    users: "Users",
+    validationRules: "Validation rules",
+    "validationRules.attributes": "Attributes of validation rules",
+    "validationRules.validationRuleGroups": "Validation rule groups of validation rules",
+    validationRuleGroups: "Validation rule groups",
+    "validationRuleGroups.attributes": "Attributes of validation rule groups",
+};
+
+const IncludeExcludeSelectionStep: React.FC<IncludeExcludeSelectionStepProps> = ({
+    d2,
+    syncRule,
+    onChange,
+}) => {
+    const [modelSelectItems, setModelSelectItems] = useState<Array<ModelSelectItem>>([]);
+    const [models, setModels] = useState<Array<typeof D2Model>>([]);
```

Comment:
[minor] Type definitions could be ``ModelSelectItem[]`` and ``(typeof D2Model)[]``. I believe tokland and I are using this in favor of ``Array<>``.

Reply:
ok I'll do it